document/FileFormat.mdk

```diff
@@ -3023,6 +3029,11 @@ or is such that the analysis might be incomplete but the results that were gener
 
 If the `level` property is absent, it shall be considered equivalent to the value `"warning"`.
 
+### `threadId` property { #notification-threadId }
+
+A `notification` object may contain a property named `threadId` whose value is an integer which
+identifies the thread which generated this notification.
```

Comment:
the thread 'associated' with this notification is better. right? the thread generating the notification could be a dedicated thread that's processing error messages produced across the app.